TheFoodieNetwork/app/src/main/java/com/fbu/thefoodienetwork/activities/ProfileActivity.java

```diff
@@ -0,0 +1,119 @@
+import com.parse.ParseFile;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+//TODO change LoginActivity goMainActivity when finish
+public class ProfileActivity extends AppCompatActivity {
+    private final static String TAG = "ProfileActivity";
+    private ActivityProfileBinding binding;
+    private ParseUser user = ParseUser.getCurrentUser();
+    private ReviewAdapter reviewAdapter;
+    private List<ParseReview> allReviews;
+    private LinearLayoutManager layoutManager;
+    private RecyclerView reviewRecyclerView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        binding = ActivityProfileBinding.inflate(getLayoutInflater());
+        View view = binding.getRoot();
+        setContentView(view);
+
+        reviewRecyclerView = binding.postsRecyclerView;
+        allReviews = new ArrayList<>();
+        reviewAdapter = new ReviewAdapter(this, allReviews);
+        reviewRecyclerView.setAdapter(reviewAdapter);
+        layoutManager = new LinearLayoutManager(this);
+        reviewRecyclerView.setLayoutManager(layoutManager);
+
+        queryReviews();
+
+        binding.editButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                goEditProfileActivity();
+            }
+        });
+
+        try {
+
+            if (user.getString(UserKeys.screenName) == null || user.getString(UserKeys.screenName).trim().equals("")) {
+                binding.textViewScreenName.setText(user.getUsername());
+            } else {
+                binding.textViewScreenName.setText(user.getString(UserKeys.screenName));
+                binding.textViewUsername.setText(user.getUsername());
+            }
+
+
+            try {
+                binding.bioTextView.setText(user.getString(UserKeys.bio));
+            } catch (Exception e) {
+                Log.i(TAG, "error:" + e.toString());
+            }
+
+            ParseFile image = user.getParseFile(UserKeys.profileImage);
+            if (image != null) {
+                Glide.with(this).load(image.getUrl()).centerCrop().circleCrop().into(binding.profileImage);
+            } else {
+                Glide.with(this).load(R.drawable.launcher1).circleCrop().into(binding.profileImage);
+            }
+
+        } catch (Exception e) {
+            Log.e(TAG, "Error: " + e);
+        }
+
+    }
+
+    private void goEditProfileActivity() {
+        Intent editIntent = new Intent(ProfileActivity.this, EditProfileActivity.class);
+        startActivity(editIntent);
+    }
+
+    private void queryReviews() {
+        ParseQuery<ParseReview> query = ParseQuery.getQuery(ParseReview.class);
+        query.include(ParseReview.AUTHOR_KEY);
+        query.include(ParseReview.RESTAURANT_KEY);
+        query.whereEqualTo(ParseReview.AUTHOR_KEY, user);
+        query.setLimit(10);
```

Comment:
10 should be moved to it's own constant named something to the effect of `REVIEW_QUERY_LIMIT`.